teradatalabs/mapr52-hive/files/supervisord.conf

```diff
@@ -0,0 +1,22 @@
+[supervisord]
+logfile = /var/log/supervisord.log
+logfile_maxbytes = 50MB
+logfile_backups=10
+loglevel = info
+pidfile = /var/run/supervisord.pid
+nodaemon = true
+directory = /tmp
+strip_ansi = false
+
+[unix_http_server]
+file = /tmp/supervisor.sock
+
+[rpcinterface:supervisor]
+supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
+
+[supervisorctl]
+serverurl = unix:///tmp/supervisor.sock
+prompt = cdh-pseudo-distributed
```

Comment:
If we do end up keeping this config maybe we should update this to say something other than cdh-pseudo-distributed?